<?xml version="1.0" encoding="UTF-8"?>
<!--
/****************************************************************************
**
** This file is part of the Blackfriars Trading Terminal
**
** Copyright (C) 2021 Alexander Mishurov
**
** GNU General Public License Usage
** This file may be used under the terms of the GNU
** General Public License version 3. The licenses are as published by
** the Free Software Foundation and appearing in the file LICENSE.GPL3
** included in the packaging of this file. Please review the following
** information to ensure the GNU General Public License requirements will
** be met: https://www.gnu.org/licenses/gpl-3.0.html.
**
****************************************************************************/
-->
<ui version="4.0">
<class>MainWindow</class>
<widget class="QMainWindow" name="mainWindow">
    <property name="geometry">
        <rect>
            <x>0</x>
            <y>0</y>
            <width>700</width>
            <height>500</height>
        </rect>
    </property>
    <property name="windowTitle">
        <string>Blackfriars</string>
    </property>

    <!-- Central Widget -->
    <widget class="CentralWidget" name="centralwidget"/>

    <!-- Menu -->
    <widget class="QMenuBar" name="menubar">
        <widget class="QMenu" name="menuFile">
            <property name="title">
                <string>File</string>
            </property>
            <addaction name="actionPref"/>
            <addaction name="actionQuit"/>
        </widget>
        <widget class="QMenu" name="menuView">
            <property name="title">
            <string>View</string>
            </property>
            <addaction name="action1View"/>
            <addaction name="action3View"/>
        </widget>
        <widget class="QMenu" name="menuHelp">
            <property name="title">
            <string>Help</string>
            </property>
            <addaction name="actionSource"/>
            <addaction name="actionAbout"/>
        </widget>
        <addaction name="menuFile"/>
        <addaction name="menuView"/>
        <addaction name="menuHelp"/>
    </widget>

    <!-- <widget class="QStatusBar" name="statusbar"/> -->

    <!-- File -->
    <action name="actionPref">
        <property name="text">
            <string>Preferences</string>
        </property>
    </action>
    <action name="actionQuit">
        <property name="text">
            <string>Quit</string>
        </property>
        <property name="shortcut">
            <string>Ctrl+Q</string>
        </property>
    </action>

    <!-- View -->
    <actiongroup name="actionGroupViews">
        <property name="exclusive">
            <bool>true</bool>
        </property>
        <action name="action1View">
            <property name="text">
                <string>Single View</string>
            </property>
            <property name="checkable">
                <bool>true</bool>
            </property>
        </action>
        <action name="action3View">
            <property name="text">
                <string>Triple View</string>
            </property>
            <property name="checkable">
                <bool>true</bool>
            </property>
        </action>
    </actiongroup>

    <!-- Help -->
    <action name="actionSource">
        <property name="text">
            <string>Source Code</string>
        </property>
    </action>
    <action name="actionAbout">
        <property name="text">
            <string>About</string>
        </property>
    </action>
</widget>
<includes>
    <include location="local">frontend/mainwindow/centralwidget.h</include>
</includes>
<resources/>
<connections/>
</ui>
